src/pages/User/Manager/index.js

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import { withRouter } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { connect } from "react-redux";
+import { bindActionCreators } from 'redux';
+
+import { redirectValueUsuario } from '../../Gerenciar/Produto/ProdutoRedux/action';
+import { getUsers } from '../../../services/usuario';
+import ManagerContainer from '../../../containers/User/Manager';
+import { compose, isEmpty, path, pathOr } from 'ramda';
+
+const Manager = ({
+  auth,
+  history,
+  redirectValueUsuario,
+}) => {
+  const [data, setData] = useState([])
+  const [loading, setLoading] = useState(true)
+  const [shouldRequest, setShouldRequest] = useState(true)
+
+  useEffect(() => {
+    if (shouldRequest) {
+      getAllUsers()
```

Comment:
Se não passar uma query com total o back-end define o limit da busca como 25
```suggestion
      getAllUsers({ total: 10 })
```